app/src/main/java/com/example/vshcheglov/webshop/data/enteties/mappers/ProductEntityDataMapper.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.vshcheglov.webshop.data.enteties.mappers
+
+import com.example.vshcheglov.webshop.data.enteties.ProductEntity
+import com.example.vshcheglov.webshop.domain.Product
+
+object ProductEntityDataMapper {
```

Comment:
As an example:

`interface RealmMapper<Model, Realm> {
    fun fromRealm(realmObject: Realm): Model
    fun toRealm(modelObject: Model): Realm
}

interface NetworkMapper<in From, out To> {
    fun mapFromNetwork(objectFrom: From): To?
}`